src/new-client/src/App/LiveRates/Tiles/TilePanel.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { memo } from "react"
+import { PriceControls } from "./PriceControls/PriceControls"
+import { NotionalInput } from "./Notional"
+import { AnalyticsTileChart } from "./AnalyticsTile/AnalyticsTileChart"
+import { TileHeader } from "./TileHeader"
+import {
+  MainTileStyle,
+  LineChartWrapper,
+  PriceControlWrapper,
+  MainTileWrapper,
+  NotionalInputWrapper,
+  TileBodyWrapper,
+} from "./responsiveWrappers"
+import { useCurrencyPairs } from "services/currencyPairs"
+import { usePrice, useHistoricalPrices } from "services/tiles"
+import { format } from "date-fns"
+import { TileView, useSelectedTileView } from "services/tiles"
+import styled from "styled-components/macro"
+
+const PanelItem = styled.div`
+  flex-grow: 1;
+  flex-basis: 20rem;
+`
+
+interface Props {
+  id: string
+}
+
+export const TilePanel: React.FC<Props> = memo(({ id }) => {
+  const currentView = useSelectedTileView()
+  const isAnalytics = currentView === TileView.Analytics
+  const currencyPairs = useCurrencyPairs()
+  const currencyPair = currencyPairs[id]
```

Comment:
It would be better to pass the `currencyPair` through props... Either that or have a `useCurrencyPair` hook that accepts the `id` and returns the currencyPair of that id only when there are relevant changes to the given currenyPair.